src/systemctl/systemctl-show.c

```diff
@@ -771,6 +773,10 @@ static void print_status_info(
                                 printf("%speak: %s", prefix, FORMAT_BYTES(i->memory_peak));
                                 prefix = " ";
                         }
+                        if (i->memory_swap_peak != CGROUP_LIMIT_MAX) {
+                                printf("%sswap peak: %s", prefix, FORMAT_BYTES(i->memory_swap_peak));
```

Comment:
I consider only showing this only if memory swap peak is greater than zero. Thoughts?

Reply:
I think this is the only place where I am pretty convinced that it is a good idea to not show the value if it is zero.